interpreter/cling/lib/Interpreter/InterpreterCallbacks.cpp

```diff
@@ -239,6 +241,27 @@ namespace cling {
     return false;
   }
 
+  void InterpreterCallbacks::CreateCodeCompleteConsumer(Interpreter* child) const {
+    ClingCodeCompleteConsumer* consumer = new ClingCodeCompleteConsumer(
+                m_Interpreter->getCI()->getFrontendOpts().CodeCompleteOpts,
```

Comment:
I think for an interpreter it might be nice to tweak the default `CodeCompleteOpts` a bit. E.g. `IncludeBriefComments` could be set to 1 in CIFactory, where we create the `CompilerInstance`? But I don't know whether this looks nice, up to you!